contrib/asff.tpl

```diff
@@ -30,7 +34,7 @@
                 "Label": "{{ $severity }}"
             },
             "Title": "Trivy found a vulnerability to {{ .VulnerabilityID }} in container {{ $target }}",
-            "Description": {{ escapeString $description | printf "%q" }},
+            "Description": {{ $description }},
```

Comment:
@ben-harvey 
`escapeString` was removed, but i catch an error now when try to handle the result:
```sh
$ cat result.asff | jq '.Findings'
parse error: Invalid string: control characters from U+0000 through U+001F must be escaped at line 22832, column 13
```

it looks like a mistake, right?

Reply:
oh, missed. it was moved. sorry. understood